ILTransform/ILRewriter.cs

```diff
@@ -378,7 +378,8 @@ private void RewriteFile(string source)
                         if (TestProject.TryGetILTypeName(source, lines, i, out string? className))
                         {
                             string qualifiedClassName = _testProject.DeduplicatedNamespaceName + "." + className!;
-                            for (int s = lineIndex; s < lines.Count; s++)
+                            // There could be a class usage before the namespace line
+                            for (int s = 0; s < lines.Count; s++)
```

Comment:
The logic for finding `NamespaceLine` is a bit odd.. it looks like it is trying to find the beginning of the method/type that the entry point is in, but then we extend it to the end of the file.  It's not probably necessary to do anything about it, but it's just inconsistent to include extra things after the entry point but not before it.

Since `NamespaceLine` can be in the middle of the file, this change definitely makes sense.

I don't believe the `s != i` is needed anymore.  `ReplaceIdent(..., IdentKind.TypeUse)` should be fine to avoid renaming the class definition itself.  (and technically line-based is probably not the correct way anyway, though it seems pretty reasonable to believe that our tests have newlines after the .class start.